README.md

```diff
@@ -1,12 +1,12 @@
 # ape-template
 
-TODO: Description
+The ape-template plugin allows you to use cookiecutter to template an ape project.
 
 ## Dependencies
 
-* [python3](https://www.python.org/downloads) version 3.7 or greater, python3-dev
-* [cookiecutter](https://cookiecutter.readthedocs.io/en/2.0.2/) version 1.7.3
-* [eth-ape](https://docs.apeworx.io/ape/stable/) 0.2.2 or greater, 
+* [python3](https://www.python.org/downloads) version 3.7.2 or greater, python3-dev
+* [cookiecutter](https://cookiecutter.readthedocs.io/en/2.0.2/) version 2.1.1
```

Comment:
It seems redundant to re-list `install_requires` items (from `setup.py`) here (both `eth-ape` and `cookiecutter`).